Sources/BarcodeHeroCore/BHBarcodeOptions.swift

```diff
@@ -7,11 +7,25 @@
 //
 
 import CoreGraphics
-import Foundation
+import UIKit
```

Comment:
I'd like to support macOS for BarcodeHero, just because it's fairly simple to catch both as well.

In order to do so, we can't use `UIKit` at all in `BarcodeHeroCore`--or, when we do, we need to provide alternatives.

The initializers I have for `CGColor` there are the initializers that work across iOS, macOS, and watchOS for Barcode generation. There are a number of other CGColor initializers, but some don't work on iOS, some don't work on macOS, etc. New versions introduced an `srgb` initializer, but it's only for the latest and greatest.

I'd suggest that we make an extension like this (avoid public if possible, but not sure if it'll error or not):

```swift
extension CGColor {
    static var safeBlack: CGColor {
        return CGColor(colorSpace: ...) // fill in with the examples
    }

    static var safeWhite: ...
}
```

Names are up in there, but at least in that case, we can re-use them over and over and not worry too much about the compatibility.